grails-app/assets/stylesheets/index/style.css

```
@@ -66,39 +66,50 @@ body {
   color: #000000;
 }
 
-#method {
-  margin-top: 10px;
-}
-
-#payerId {
-  margin-top: 10px;
-  margin-bottom: 10px;
-}
-
 .form-control input {
   border: 2px solid black;
   color: black;
   border-radius: 10px;
-  margin-bottom: 10px;
+  margin-bottom: 20px;
   padding: 10px;
   font-size: 14px;
   width: 100%;  
 }
 
 .form-control i {
   position: absolute;
-  top: 45px;
+  top: 35px;
   right: 10px;
   visibility: hidden;
 }
 
 .form-control small {
   position: absolute;
   bottom: 0;
-  left: 0;
+  left: 15px;
   visibility: hidden;
 }
 
+.form-control #method {
+  border: 2px solid black;
+  color: black;
+  border-radius: 10px;
+  margin-bottom: 25px;
+  padding: 10px;
+  font-size: 14px;
+  width: 100%;  
+}
+
+.form-control #payerId {
```

Dá pra unificar, validar outros pontos do PR
```suggestion
.form-control #method,
.form-control #payerId {
```